content/en/docs/tasks/tools/included/kubectl_installs_gcloud.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "gcloud kubectl install"
+description: "How to install kubectl with gcloud snippet for inclusion in each OS-specific tab."
+toc_hide: true
+---
```

Comment:
This also looks like it might be intended to work like a Hugo [headless bundle](https://gohugo.io/content-management/page-bundles/#headless-bundle).

Reply:
Yes - updated in https://github.com/kubernetes/website/pull/26682/commits/62ffd9d9a4e0fa6bac949ee8dfcfe12130afb357